cmd/protoc-gen-go-grpc/grpc.go

```diff
@@ -36,9 +36,11 @@ const (
 
 type serviceGenerateHelperInterface interface {
 	formatFullMethodName(service *protogen.Service, method *protogen.Method) string
+	generateNewClientConnInterface(g *protogen.GeneratedFile, clientName string)
```

Comment:
Why "New"?

And it's generating a client struct, not the ClientConn interface, isn't it?  So `generateClientStruct`?

Reply:
Done.